packages/orbit-components/src/BadgeList/BadgeList.stories.js

```diff
@@ -0,0 +1,104 @@
+// @flow
+import * as React from "react";
+import { action } from "@storybook/addon-actions";
+import { text, select } from "@storybook/addon-knobs";
+
+import * as Icons from "../icons";
```

Comment:
Generally that's true, but for stories should be okay, because this module is not being imported (outside storybook). In the static bundle for Storybook they are present there anyway already 🙂

Reply:
Oh, right.